tests/vereya/test_consistency.py

```diff
@@ -0,0 +1,136 @@
+import unittest
+import logging
+import json
+import time
+from tagilmo import VereyaPython
+import tagilmo.utils.mission_builder as mb
+from tagilmo.utils.vereya_wrapper import MCConnector, RobustObserver
+from base_test import BaseTest
+
+
+def init_mission(mc, start_x=None, start_y=None, start_z=None):
```

Comment:
if this function has nothing specific you can replace it with init_mission from common.py

Reply:
@noskill  it does differ